src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -150,7 +151,7 @@ class java_bytecode_parsert:public parsert
         error() << "unexpected end of bytecode file" << eom;
         throw 0;
       }
-      result<<=8;
+      result <<= CHAR_BIT;
       result|=in->get();
```

Comment:
A question for the more Java orientated folks : is this fair or should this follow the JVM standard which, IIRC, says something about the number of bits in each type.